NetIO/CryptIO.cpp

```diff
@@ -57,14 +57,14 @@ void DS::GenPrimeKeys(uint8_t* K, uint8_t* N)
     init_rand();
 
     while (BN_num_bytes(bn_key) != 64) {
-        BN_generate_prime(bn_key, 512, 1, 0, 0, 0, 0);
+        BN_generate_prime_ex(bn_key, 512, 1, nullptr, nullptr, nullptr);
```

Comment:
These days it uses a mix of both...  Since `nullptr` has been around since at least GCC 4.6, I'd like to start preferring it in new/refactored code.